05/app.js

```diff
@@ -1,53 +1,73 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+  paragraphs: {
+    'p1': 0,
+  },
+  links: {
+    '/dolor.html': 0,
+  }
 };
 
 /* tutaj umieść swój kod */
+const pElements = document.querySelectorAll('p');
+
+const handleClick = function(e) {
+
+  if (e.target.tagName === 'P') {
+  const id = e.target.dataset.id;
+  // update the paragrafs
+  stats.paragraphs[id] ? stats.paragraphs[id]++ : stats.paragraphs[id] = 1;
+} else if (e.target.tagName === 'A') {
+  // stop linking
+  e.preventDefault();
```

Comment:
Nie do końca, jak klikasz w `<a>` to również klikasz w `<p>`.
W takim wypadku statystyki dla obu powinny się zwiększać.